domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/StationWithParentStationNotice.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class StationWithParentStationNotice extends ErrorNotice {
+
+    public StationWithParentStationNotice(final String childId,
+                                          final String parentId) {
+        super("stops.txt", E_037,
+                "Non null `parent_station`",
+                "Stop with id:`" + childId + " is a station (location_type=1) with parent_id:`" +
+                        parentId + "`. This is forbidden. Expected null",
```

Comment:
Suggest a slightly clearer message here:

```suggestion
                "Stop with id:`" + childId + " is a station (location_type=1) with parent_id:`" +
                        parentId + "`. Stations cannot have parent stops. Expected null",
```